src/libs/network/endpointHandlers.ts

```
@@ -389,6 +396,12 @@ export default class ServerHandlers {
                 }
                 result.response = nativeBridgeResult
                 break
+
+            case "l2ps_hash_update":
+                var l2psHashResult = await ServerHandlers.handleL2PSHashUpdate(tx)
+                result.response = l2psHashResult
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)